### DISCUSSION TITLE
Not able to test the project although have a valid API keys and tokens.

### DISCUSSION THREAD

**Discussion by ajayjadhav900** [langroodi/Adaptive-AUTOSAR#35] [Q&A] [answered]
I have clone , build the entire repo and is built successfully.
./build/adaptive_autosar ./configuration/execution_manifest.arxml ./configuration/extended_vehicle_manifest.arxml ./configuration/diagnostic_manager_manifest.arxml ./configuration/health_monitoring_manifest.arxml

there is one typo error - **adaptive_autosar exe is present in build/bin directory.**

i have connected volvo page and configure the API and token
still getting this error
StateManagement Context ID:Lifetime;Context Description:Application lifetime logs;Log Level:Info;EM initial machine state transition is fetched successfully.
Mon May 6 11:47:59 2024
StateManagement Context ID:Lifetime;Context Description:Application lifetime logs;Log Level:Info;EM is transited to the start-up state successfully.
Mon May 6 11:47:59 2024
StateManagement Context ID:Lifetime;Context Description:Application lifetime logs;Log Level:Info;Execution state is reported successfully.
Mon May 6 11:48:01 2024
ExtendedVehicle Context ID:Lifetime;Context Description:Application lifetime logs;Log Level:Error;**Setting the VIN failed due to unexpected RESTful response format.**

I have crossed checks ,API keys and token are valid.

**Please assist on this.**

**Comment by langroodi** [comment, marked as answer]
Volvo has updated to V2 while the Extended Vehicle is on V1 which may causes this error. Currently, there is no plan to update toward version 2.

**Comment by ajayjadhav900** [comment]
closed
